### PR TITLE
add English documentations for 3 num_filter functions

### PR DESCRIPTION
**Pull request by kumagi**
Add documentation of three number normalization filters.

### CODE REVIEW

**Comment by kumagi** [comment]
I did #177

File: source/en/fv_convert.rst [hunk 1]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
 
   :value:  Specifies value to add. For example, if we add 3 to the original value, we use "3". Note that it is not numeric but a string. It is treated as a floating-point number internally.
 
+.. describe:: linear_normalization
+
+ It normalizes linearly value between 0 to 1.
+ It requires two arguments "min" and "max", and these values cannot be omited.
+ It transform given value x to be between 0 to 1 with formula (x-min) / (max - min).
```
Comment (gwtnb) [inline source/en/fv_convert.rst]: transforms

File: source/en/fv_convert.rst [hunk 2]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+ If the x is smaller than "min", it truncate value to be 0.
```
Comment (gwtnb) [inline source/en/fv_convert.rst]: it is truncated to 0.

File: source/en/fv_convert.rst [hunk 3]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+ If the x is bigger than "max", it truncate value to be 1.
```
Comment (gwtnb) [inline source/en/fv_convert.rst]: same.

File: source/en/fv_convert.rst [hunk 4]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+ These truncation behavior is switched by "truncate" option.
+ If you give argument which "min" is bigger than "max", the invalid_parameter exception will be raised and fail to create converter.
```
Comment (gwtnb) [inline source/en/fv_convert.rst]: If "min" is greater than "max", ...

File: source/en/fv_convert.rst [hunk 5]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
 
   :value:  Specifies value to add. For example, if we add 3 to the original value, we use "3". Note that it is not numeric but a string. It is treated as a floating-point number internally.
 
+.. describe:: linear_normalization
+
+ It normalizes linearly value between 0 to 1.
+ It requires two arguments "min" and "max", and these values cannot be omited.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: omited -> omitted

**Comment by gwtnb** [comment]
You should use spell checker.

File: source/en/fv_convert.rst [hunk 6]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+    "num_filter_types" : {
+        "zero_to_hundred": { "method": "linear_normalization", "min": "0", "max":"100" }
+    },
+    "num_filter_rules" : [
+        {"key" : "*", "type": "zero_to_hundred", "suffix": "linear_normalized" }
+    ],
+
+.. describe:: gaussian_normalization
+
+ It normalize values between -1 to +1, supporsing values are ditributed on normal distribution.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: supporsing -> supposing

File: source/en/fv_convert.rst [hunk 7]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+ It requires two arguments "average" and "standard_deviation", and these values cannot be omited.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: omitted

File: source/en/fv_convert.rst [hunk 8]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+ It transform given value x to be -1 to +1 with formula (x - average) / standard_deviation.
+ For this reason, anomaly value can be less than -1 or more than +1.
+ You cannot specify negative value for "standard_deviation". It causes invalid_parameter exception.
+
+  :average:  Give average of input data. If average value is 80, you should specify like "80". Notice that it is not numeric but string type. Value is treated as doubled precision froating point value inside.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: floating

File: source/en/fv_convert.rst [hunk 9]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+    "num_filter_types" : {
+        "gaussian_80_2.3": { "method": "gaussian_normalization", "average": "80", "standard_deviation":"2.3" }
+    },
+    "num_filter_rules" : [
+        {"key" : "*", "type": "gaussian_80_2.3", "suffix": "gaussian_normalized" }
+    ],
+
+.. describe:: sigmoid_normalization
+
+ It normalize values between 0 to 1, by using sigmoid funtion.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: function

**Comment by kmaehashi** [comment]
@kumagi  @gwtnb and I commented, could you update the patch?

File: source/en/fv_convert.rst (hunk 1 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 2 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 3 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 4 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 6 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 5 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 7 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

File: source/en/fv_convert.rst (hunk 8 above)
Comment (kumagi) [inline source/en/fv_convert.rst]: thanks

**Comment by kmaehashi** [comment]
I fixed some typos and rebased to the latest develop branch.

File: source/en/fv_convert.rst [hunk 10]
```diff
@@ -259,7 +325,7 @@ Each rule is a dictionary whose keys are "key", "except" (optional), "type" and
 
  :key:       Specifies to which keys in a datum we apply the rule. For further explanation, please read counterpart in "string_filter_rules" section.
  :except:    Specifies which keys to exclude from the match. This is an optional parameter. For further explanation, please read counterpart in "string_filter_rules" section.
- :type:      Specidies a name of a filter used. This filter must de defined in "string_filter_types". No filter is available if no filter is defined in "string_filter_types".
+ :type:      Specifies a name of a filter used. This filter must de defined in "string_filter_types". No filter is available if no filter is defined in "string_filter_types".
```
Comment (gintenlabo) [inline source/en/fv_convert.rst]: must de defined

File: source/en/fv_convert.rst [hunk 11]
```diff
@@ -243,6 +243,73 @@ The followings are available values of "method" and keys that must be specified.
...
+    "num_filter_types" : {
+        "zero_to_hundred": { "method": "linear_normalization", "min": "0", "max":"100" }
+    },
+    "num_filter_rules" : [
+        {"key" : "*", "type": "zero_to_hundred", "suffix": "linear_normalized" }
+    ],
+
+.. describe:: gaussian_normalization
+
+ It normalize values between -1 to +1, supposing values are distributed on normal distribution.
```
Comment (gintenlabo) [inline source/en/fv_convert.rst]: normalizes?

**Pull request merged**